src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/InnvilgeBarnetilsyn.tsx

```diff
@@ -118,7 +119,9 @@ export const InnvilgeBarnetilsyn: React.FC<Props> = ({ lagretVedtak, barnIBehand
                 `/api/sak/vedtak/tilsyn-barn/${behandling.id}/beregn`,
                 'POST',
                 vedtaksRequest
-            ).then(settBeregningsresultat);
+            )
+                .then(settBeregningsresultat)
+                .finally(() => settLaster(false));
```

Comment:
Hmm, ser at laster ikke ble så enkelt. Denne `laster` brukes jo både ved lagring og beregning, men de burde egentlig være to forskjellige så beregning sin laster ikke clearer lagring sin midt i en request

Reply:
Da beholder jeg det foreløpig kun på lagring og så får vi vurdere om vi trenger det på beregning senere... 😢